tools/ci_build/build.py

```diff
@@ -1467,33 +1467,38 @@ def build_nuget_package(configs, use_cuda, use_openvino, use_tensorrt, use_dnnl,
     else:
         pass
 
+    # set build directory based on build_dir arg
+    native_dir = os.path.normpath(os.path.join(source_dir, build_dir))
```

Comment:
why is build_dir appended to source_dir? what if build_dir is an absolute path?

Reply:
If a component is an absolute path, all previous components are thrown away and joining continues from the absolute path component.